test/image/gif.h

```diff
@@ -0,0 +1,41 @@
+#include <cxxtest/TestSuite.h>
+
+#if defined(HAVE_CONFIG_H)
+#include "config.h"
+#endif
+
+#include "common/memstream.h"
+#include "image/gif.h"
+#include "graphics/surface.h"
+
+class GIFDecoderTestSuite : public CxxTest::TestSuite {
+#ifdef USE_GIF
```

Comment:
OK, this fails on platforms without USE_GIF because the test is nonexistent anymore.
The cxxtestgen.py script doesn't track #ifdef and when it generates a runTests method, it tries to use this test. This obviously fails.
Like in test/common/array.h for USE_CXX11, I think it's simpler to move the #ifdef/#endif inside the function.

Reply:
Sorry for the trouble - should be fixed now.

Reply:
Working now.